scripts/portstat

```diff
@@ -104,30 +118,36 @@ class Portstat(object):
         """
         # Get speed from APPL_DB
         full_table_id = PORT_STATUS_TABLE_PREFIX + port_name
-        speed = self.db.get(self.db.APPL_DB, full_table_id, PORT_SPEED_FIELD)
-        if speed is None:
-            speed = PORT_RATE
-        else:
-            speed = int(speed)//1000
+        speed = PORT_RATE
+        for ns in self.multi_asic.get_ns_list_based_on_options():
+            self.multi_asic.connect_config_db_for_ns(ns)
+            speed = self.db.get(self.db.APPL_DB, full_table_id, PORT_SPEED_FIELD)
+            if speed is not None:
+                return int(speed)//1000
```

Comment:
wondering why can't the decorator be used here. It would be good to be consistent. You can pass the namespace arg to the API.

Reply:
This file was added to this PR by mistake. Removed the changes to this PR